framework/sandstone_tests.cpp

```diff
@@ -79,51 +79,42 @@ SandstoneTestSet::SandstoneTestSet(struct test_set_cfg cfg, unsigned int flags)
         struct test_cfg_info ti;
         ti.status = test_cfg_info::enabled;
         ti.test = &test;
-        test_map[test.id] = ti;
         test_set.push_back(&test);
     }
 };
 
-struct test_cfg_info SandstoneTestSet::enable(struct test *t) {
-    struct test_cfg_info &ti = test_map[t->id];
+struct test_cfg_info SandstoneTestSet::append(test_cfg_info t)
```

Comment:
I was thinking that maybe `add()` would be a better name. I know it adds it to the end and therefore appends, but add/remove dichotomy is more cleaner in this case, IMO.

Reply:
Ok.